app/controllers/api/v2/roles_controller.rb

```diff
@@ -51,11 +51,30 @@ def destroy
         process_response @role.destroy
       end
 
+      api :POST, "/roles/:id/clone", N_("Clone a role")
+      param :id, String, :required => true
+      param :new_name, String, :required => true, :desc => N_('Name of the cloned role')
```

Comment:
I see param `name` is used few lines below, which one is correct? also could it be enhanced so it accepts the whole role param_group? I'd like to set the organization while cloning it as well as description.